client.go

```diff
@@ -87,13 +87,15 @@ func New(address string, opts ...ClientOpt) (*Client, error) {
 	if copts.timeout == 0 {
 		copts.timeout = 10 * time.Second
 	}
-	rt := fmt.Sprintf("%s.%s", plugin.RuntimePlugin, runtime.GOOS)
+
```

Comment:
just curiosity, this change is not related one. 
the original code seems to reduce the indent and looks more clear. not sure why change that.

Reply:
It's always hard to judge code style related changes :) As for me it's more clear what used when, and we don't have always call `fmt.Sprintf` here. But I don't feel strongly about it.

Reply:
yeah. Just saw this change which is not related to your pr.